### PR TITLE
Change AssetTransactionGroup to AssetTransaction

### PR DESCRIPTION
**Pull request by sgkim126**
@joojis I'll merge it after the timelock merged.

### CODE REVIEW

**Comment by sgkim126** [comment]
This PR is related to https://github.com/CodeChain-io/codechain/pull/833

File: src/core/index.ts [hunk 1]
```diff
@@ -593,13 +593,8 @@ function checkRegistrar(registrar: PlatformAddress | string | null) {
...
 
-function checkTransactions(transactions: Transaction[]) {
-    if (!Array.isArray(transactions)) {
-        throw Error(
-            `Expected transactions param to be an array but found ${transactions}`
-        );
-    }
-    // FIXME: check all transaction are valid
+function checkTransaction(_transaction: Transaction) {
+    // FIXME: check whether the transaction aris valid
```
Comment (joojis) [inline src/core/index.ts]: typo?

File: src/core/index.ts [hunk 2]
```diff
@@ -112,18 +112,18 @@ export class Core {
     }
 
     /**
-     * Creates AssetTransactionGroup action which can mint or transfer assets through
+     * Creates AssetTransaction action which can mint or transfer assets through
      * AssetMintTransaction or AssetTransferTransaction.
-     * @param params.transactions List of transaction
+     * @param params.transaction List of transaction
```
Comment (joojis) [inline src/core/index.ts]: List of transaction -> Transaction

File: src/core/index.ts (hunk 1 above)
Comment (sgkim126) [reply src/core/index.ts]: Fixed

File: src/core/index.ts (hunk 2 above)
Comment (sgkim126) [reply src/core/index.ts]: Fixed

**Comment by sgkim126** [comment]
@joojis PR updated

**Comment by sgkim126** [comment]
I updated docker tag.

**Pull request merged**